shared/build.gradle.kts

```diff
@@ -24,9 +24,12 @@ plugins {
     alias(libs.plugins.serialization)
 }
 //https://youtrack.jetbrains.com/issue/KT-66448/Multiplatform-wizards.-Get-rid-of-deprecated-kotlinOptions
+@OptIn(ExperimentalKotlinGradlePluginApi::class)
 kotlin {
+    compilerOptions {
+        freeCompilerArgs.add("-Xexpect-actual-classes")
```

Comment:
Since Expect/Actual are still in beta, without this the IDE will have some warning. 
Comment this and check `ImagePicker` expect class 

For reference, check [this](https://kotlinlang.org/docs/multiplatform-expect-actual.html#expected-and-actual-classes)